tools/who_what_benchmark/requirements.txt

```diff
@@ -1,9 +1,11 @@
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
 transformers>=4.35.2
 sentence-transformers>=2.2.2
-openvino>=2024.3.0
+openvino
+openvino-tokenizers
+openvino-genai
```

Comment:
- it's enough to install `openvino-genai` only as it depends on tokenizes and OpenVINO
- w/o `~=2025.0.0.0.dev` or `--pre` nightly will not be installed and `--extra-index-url` is not used
- could you please drop these lines except WWB installation from CI https://github.com/openvinotoolkit/openvino.genai/blob/e6a435fdf4b4c14d0c664aaf7fe1d763b341faff/.github/workflows/llm_bench-python.yml#L104-L107 to fully rely on `tools/who_what_benchmark/requirements.txt` ?

Reply:
I have reworked it, please take a look.